docs/Arguments.md

```diff
@@ -139,6 +140,10 @@ usage: -m llumnix.entrypoints.vllm.api_server [-h]
 - Possible choices: balanced, load, queue, rr
 - Default: "load"
 
+`--power-of-k-choice`
```

Comment:
I don't recommend using the term power-of-k-choices here. It's a well-established concept in the specific context of decentralized load balancing. But here it seems to me that you are only doing some sort of randomizing inside a centralized scheduler. Calling it power-of-k-choices is pretty confusing.

Reply:
Get. I will rename it to 'topk-random-dispatch'.